### PR TITLE
Gallery, projects, and fixes/touch ups

### PR DESCRIPTION
**Pull request by 0tpyrk**
New features:
- Image gallery
- Projects page
- General fixes and touch ups

### CODE REVIEW

File: portfolio/src/main/webapp/aboutme.html [hunk 1]
```diff
@@ -6,22 +6,33 @@
...
+      <div id="about-me-info">
+        <p>My name is William Lew. I am from San Ramon, CA. I go to Lewis & Clark College and am a Computer Science major.
+            My hobbies include playing video games, writing, listening to music, and coding.</p>
+        <br>
+        <img src="/images/me.jpg" alt="A photo of me">
+        <br>
+        <br>
+        <div id="logos">
+          <a href="https://github.com/0tpyrk"><img src="images/logos/github.png" alt="GitHub"></a>
+          <a href="https://www.linkedin.com/in/william-lew-382018189/"><img src="images/logos/linkedin.png" alt="Linkedin"></a>
```
Comment (Yunni) [inline portfolio/src/main/webapp/aboutme.html:32]: Please avoid lines longer than 80 characters

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -41,10 +39,134 @@ li a {
...
+  content: "";
+  display: table;
+}
+
+/* if screen is < 800px, stack as rows */
+@media screen and (max-width: 800px) {
+  .column {
+    width: 100%;
+  }
+}
```
Comment (Yunni) [inline portfolio/src/main/webapp/style.css:172]: Please add a new line at the end.

Comment (0tpyrk) [reply portfolio/src/main/webapp/style.css:172]: Got it!

File: portfolio/src/main/webapp/aboutme.html (hunk 1 above)
Comment (0tpyrk) [reply portfolio/src/main/webapp/aboutme.html:32]: Will do, thanks for the reminder.

**Pull request merged**